nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -268,3 +268,7 @@ def get_weight_name(nncf_graph: NNCFGraph, target_point: ONNXTargetPoint) -> str
     def should_quantize_weight(weight_name: str, quantized_weight_names: Set[str]) -> bool:
         # If the nodes share one weight tensor, we should have only one quantizer on that
         return weight_name not in quantized_weight_names
+
+    @staticmethod
+    def is_constant_matmul(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        return node.layer_attributes.has_weight()
```

Comment:
Not only `matmul` layers can contain the `layer_attributes` field. Refer to [nncf/onnx/graph/nncf_graph_builder.py](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/graph/nncf_graph_builder.py) find other metatypes. The name of the method is incorrect.

Reply:
Do nothing, If you see here no issues.